tests/lib/rules/block-spacing.js

```diff
@@ -240,103 +240,275 @@ ruleTester.run("block-spacing", rule, {
             output: "{foo();}",
             options: ["never"],
             errors: [
-                { type: "BlockStatement", line: 1, column: 1, messageId: "extra", data: { location: "after", token: "{" } },
-                { type: "BlockStatement", line: 1, column: 10, messageId: "extra", data: { location: "before", token: "}" } }
+                {
+                    type: "BlockStatement",
+                    line: 1,
+                    column: 2,
+                    messageId: "extra",
+                    data: { location: "after", token: "{" },
+                    endLine: 1,
+                    endColumn: 3
+                },
```

Comment:
It might be nice for readability to put `endLine` and `endColumn` after the `column` line? (the same applies to all tests).

Could you please also add some invalid `"never"` tests with multiple extra spaces, I think all tests have only 1 space.

And `endLine` + `endColumn` to some `"always"` tests.

Reply:
done